janitor/functions/pivot.py

```diff
@@ -54,6 +54,7 @@ def pivot_longer(
     [`select_columns`][janitor.functions.select.select_columns] syntax.
 
     Examples:
+
```

Comment:
In general, we don't need this newline anymore after a section heading with the new parser. The entire block just needs to be indented once, whether it's a code-block or a text block within Examples. So it can be like

```python
Examples:
    This is some text

    >>> x = 1
```

or 

```python
Examples:
    >>> x = 1
```

Reply:
I feel I'm still missing something from the styling ... local preview still failing

Reply:
@samukweku Local preview? Can you take a screenshot to show what you mean? The netlify documentation seems to look okay to me

Reply:
netlify does not show the latest ... it shows what is already merged into the repo. not sure why that is 


![image](https://user-images.githubusercontent.com/25709809/219844461-8154b732-839d-45e8-8411-11c6afe7480e.png)


Notice how the selection takes the `....` as well. I liked the initial setup where the `...` is not copied. Weirdly enuf this happens only in this PR, which makes me wonder where else I had touched to cause the break :man_shrugging:

Reply:
Huh, weird. This netlify link? https://deploy-preview-1242--pyjanitor.netlify.app/

It looks fine for me (the `...` is not selected, and the example u added in this PR shows up just fine):

![image](https://user-images.githubusercontent.com/30731072/219934486-d34f6920-37ec-4465-8adb-0f96990918ee.png)